### PR TITLE
fix(docs): update tabs and control tables

### PR DESCRIPTION
**Pull request by alizedebray**

### CODE REVIEW

File: packages/documentation/.storybook/styles/components/blocks.scss
Comment (oliverschuerch) [inline packages/documentation/.storybook/styles/components/blocks.scss:1]: This has already been fixed/released: https://design-system.post.ch
Please apply this change only if the bug still persists.

File: packages/documentation/.storybook/styles/components/tabs.scss
Comment (oliverschuerch) [inline packages/documentation/.storybook/styles/components/tabs.scss:1]: Nice!
But I still see a horizontal scrollbar.
image

**Review by oliverschuerch — CHANGES_REQUESTED** [review]

File: packages/documentation/.storybook/styles/components/tabs.scss
Comment (alizedebray) [reply packages/documentation/.storybook/styles/components/tabs.scss:1]: There is another ticket for the scrollbar: https://github.com/swisspost/design-system/issues/3225

File: packages/documentation/.storybook/styles/components/blocks.scss
Comment (alizedebray) [reply packages/documentation/.storybook/styles/components/blocks.scss:1]: The issue is still there on the released documentation: 
image

**Review by oliverschuerch — APPROVED** [review]

File: .changeset/gorgeous-frogs-build.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@swisspost/design-system-documentation': patch
+---
+
+Added missing borders to control tables and updated tabs to improve visual continuity and readability.
```
Comment (oliverschuerch) [inline .changeset/gorgeous-frogs-build.md:5]:
```suggestion
Updated tabs to improve visual continuity and readability.
```

**Review by oliverschuerch — APPROVED** [review]

**Pull request merged**